dashboard/app/models/peer_review.rb

```diff
@@ -39,32 +47,25 @@ class PeerReview < ApplicationRecord
 
   validates :status, inclusion: {in: %w{accepted rejected}}, if: -> {from_instructor}
 
-  after_update :mark_user_level, if: -> {saved_change_to_status? || saved_change_to_data?}
+  before_save :add_status_to_audit_trail, if: -> {reviewer_id? && (status_changed? || data_changed?)}
```

Comment:
These changes are just moving around "macro style" methods to align with style guide -- I missed one of the `after_save` callbacks when I was working on this b/c it was ordered weirdly.

https://github.com/rubocop/rails-style-guide#macro-style-methods